_content/news/_posts/2019-01-14-hervorming.markdown

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "Hervorming jeugdcompetitie"
+date: 2019-01-07 10:00:00
+description: Volgend seizoen gaat de jeugdcompetitie er heel anders uitzien
+cover: /news/img/2018-12-28-lizards-grid-rsz.jpg
+permalink: /news/2019-01-14-hervorming/
+---
+
+Op de Algemene Vergadering van basketbal vlaanderen werd in november 2018 de hervorming van de jeugdcompetitie goedgekeurd.
+
+Doel is een evenwichtigere competitie bij de jeugd en een goed gevulde kalender, zodat jonge spelers optimale leerkansen krijgen op het terrein. Dit zowel op het hoogste niveau, als alle niveaus eronder. Verder wil de bond op deze manier de clubs ook aanzetten tot een betere jeugdwerking en actievere recrutering.
+
+### Wat betekent dit praktisch
+
+Deze wijziging zal vooral impact hebben op de leeftijden U14 tot U21, waar de landelijke en provinciale competities gaan verdwijnen. 
+
+In plaats daarvan komen er 5 nieuwe niveaus in de competitie, waar in niveau 1 de absolute top over heel vlaanderen het tegen elkaar uitvecht en niveau 5 eerder recreatief basketbal 'in de buurt' toestaat, met nog 3 niveaus daar tussen in. 
+
+Naargelang het niveau waarin je speelt daalt wordt ook de radius waarin gespeeld wordt kleiner, maar niet meer beperkt tot de provincie grenzen. Voor het merendeel van onze ploegen zal dit concreet betekenen dat we regelmatig een wedstrijd gaan kunnen spelen in het oosten van Vlaams-Brabant en zuiden van de provincie Antwerpen en misschien wat minder in het noorden van Limburg.
```

Comment:
Naargelang het niveau waarin je speelt wordt de radius waarin je speelt aangepast: hoe lager het niveau, hoe kleiner de radius. Er wordt geen rekening meer gehouden met de provincies. De details vind je in de link beneden. Voor het merendeel van onze ploegen zal dit betekenen dat we regelmatig een wedstrijden zullen spelen in het oosten van Vlaams-Brabant en het zuiden van de provincie Antwerpen en misschien wat minder in het noorden van Limburg.